packages/forma-36-website/src/pages/components/select-field/index.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Select Field'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/FormLabel'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-formlabel--default'
+---
+
+Select Field is used for choosing a single option from a list of options.
+
+```jsx
+<React.Fragment>
```

Comment:
Tiny nitpick, but you don't need to use `React.Fragment` if at the root we're just rendering one element/component. You can just directly use `<SelectField>[...blahblahblah]</SelectField>`